src/__tests__/patients/diagnoses/AddDiagnosisModal.test.tsx

```diff
@@ -43,75 +41,66 @@ describe('Add Diagnosis Modal', () => {
     jest.spyOn(PatientRepository, 'find').mockResolvedValue(patient)
     jest.spyOn(PatientRepository, 'saveOrUpdate').mockResolvedValue(patient)
 
-    const wrapper = mount(
-      <AddDiagnosisModal patient={patient} show onCloseButtonClick={onCloseSpy} />,
-    )
-
-    wrapper.update()
-    return { wrapper }
+    return render(<AddDiagnosisModal patient={patient} show onCloseButtonClick={onCloseSpy} />)
   }
   beforeEach(() => {
     console.error = jest.fn()
   })
   it('should render a modal', () => {
-    const { wrapper } = setup()
+    setup()
 
-    const modal = wrapper.find(Modal)
+    expect(screen.getByRole('dialog')).toBeInTheDocument()
 
-    expect(modal).toHaveLength(1)
-
-    const successButton = modal.prop('successButton')
-    const cancelButton = modal.prop('closeButton')
-    expect(modal.prop('title')).toEqual('patient.diagnoses.new')
```

Comment:
This assertion can be achieved I guess

Reply:
> @proful This assertion can be achieved I guess
This looks like the RTL equavilant, which Emma added
   ```js
    expect(screen.getByRole('button', { name: /patient\.diagnoses\.new/i })).toBeInTheDocument()
    ```